doc/PROTOCOL-WEB.md

```diff
@@ -61,9 +62,17 @@ Message framing (vs. [http2-transport-mapping](http://www.grpc.io/docs/guides/wi
 
 1. Response status encoded as part of the response body
   * Key-value pairs encoded as a HTTP/1 headers block (without the terminating newline).
+  ```
```

Comment:
consider adding a link to the actual ref:
https://tools.ietf.org/html/rfc7230#section-3.2